File: C/1010.c
#include <stdio.h>

int main() {
    int c1, c2, n1, n2;
    float m1, m2;

    scanf("%d %d %f", &c1, &n1, &m1);
    scanf("%d %d %f", &c2, &n2, &m2);
    
    printf("VALOR A PAGAR: R$ %.2f\n", (n1 * m1) + (n2 * m2));
    return 0;
}

File: 1010/1010.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner scanner = new Scanner(System.in);
		
		scanner.next();
		int n1 = scanner.nextInt();
		float m1 = scanner.nextFloat();
		
		scanner.next();
		int n2 = scanner.nextInt();
		float m2 = scanner.nextFloat();
		
		System.out.println(String.format("VALOR A PAGAR: R$ %.2f", (n1 * m1) + (n2 * m2))); 
	}
}